build.gradle

```diff
@@ -51,13 +49,13 @@ buildscript {
  * 4. Comment 'wrapper' task
  */
 //task wrapper(type: Wrapper) {
```

Comment:
Righto! Took it out entirely. Sounds like we can do future upgrades either via command line argument or simply tweaking `gradle-wrapper.properties`